src/components/BoardGrid/BoardGridContainer.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { createBoard, getUserBoards } from '../../actions/boards/actions';
+import { selectUserBoards } from '../../selectors/boardSelectors';
+import { getUserId } from '../../selectors/authSelectors';
+import BoardGrid from './BoardGrid';
+
+const BoardGridContainer = () => {
+
+  const userBoards = useSelector(selectUserBoards);
+  const [boards, setBoards] = useState(userBoards)
+  const userId = useSelector(getUserId);
+  const dispatch = useDispatch();
+  const createNewBoard = (boardData) => dispatch(createBoard(boardData));
+  // const getBoards = () => dispatch(getUserBoards());
+
+  useEffect(() => {
+    dispatch(getUserBoards());
+  }, [boards, setBoards, dispatch]);
+
+  return (
+    <BoardGrid userBoards={userBoards} userId={userId} createNewBoard={createNewBoard} />
+  )
+}
```

Comment:
Do you really need `boards` here? As I see, you just get userBoards, sets them to boards and pass to child component